test/commands/git/sync_track_test.rb

```diff
@@ -4,11 +4,21 @@ class Git::SyncTrackTest < ActiveSupport::TestCase
   test "no change when git sync SHA matches HEAD SHA" do
     track = create :track, slug: 'fsharp', synced_to_git_sha: "HEAD"
 
+    Git::SyncConceptExercise.expects(:call).never
     Git::SyncTrack.(track)
 
     refute track.changed?
   end
 
+  test "resyncs when force_sync is passed" do
+    track = create :track, slug: 'fsharp', synced_to_git_sha: "HEAD"
+
+    Git::SyncConceptExercise.expects(:call).at_least_once
```

Comment:
Perhaps also verify that the concepts are synced?

```suggestion
    Git::SyncConceptExercise.expects(:call).at_least_once
    Git::SyncConcept.expects(:call).at_least_once
```

And perhaps add a commented line for the practice exercise syncing?